mach_eval/analyzers/electromagnetic/flux_linkage_analyzer.py

```diff
@@ -0,0 +1,141 @@
+import os
+import pandas as pd
+import numpy as np
+from time import time as clock_time
+
+class FluxLinkageJMAG_Problem:
+    """Problem class for processing FEA flux linkages
+    Attributes:
+        toolJmag: JMAG tool required for FEA scripting
+        phase_names: names of phases in machine
+        rated_current: rated current of machine
+    """
+
+    def __init__(self, toolJmag, phase_names, rated_current):
+        self.toolJmag = toolJmag
+        self.phase_names = phase_names
+        self.rated_current = rated_current
+    
+    def add_em_study(self, study_name):
```

Comment:
Having the analyzer call this function doesn't seem right. The analyzer shouldn't need to be providing a study name, or even thinking about making studies.

But this could be an internal helper function that the problem class's `run()` function uses.